### PR TITLE
General styling changes before pushing to heroku

### PR DESCRIPTION
**Pull request by sdmotto**
- Fixed duplicate nav bar
- Updated server searching and adding pages
- Implemented fuzzy searching for servers

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -30,14 +30,14 @@ def create
   def list
     if params[:server_name]
       @search = params[:server_name]
-      @found_games = Game.where(name: params[:server_name])
+      @found_games = Game.where("LOWER(name) LIKE ?", "%#{@search.downcase}%")
```
Comment (mechabubba) [inline app/controllers/games_controller.rb:33]: don't think we should be doing inline sql, but its fine for now.

Comment (mechabubba) [reply app/controllers/games_controller.rb:33]: it does work... do not touch it. lol

**Comment by mechabubba** [comment]
looks good to me :D

**Comment by alexlehman99** [comment]
good looks kings

**Comment by mechabubba** [comment]
a3abce6 goated commit

**Pull request merged**